test/conformance/enqueue/urEnqueueMemImageCopy.cpp

```diff
@@ -1,2 +1,4 @@
 // Copyright (C) 2023 Intel Corporation
 // SPDX-License-Identifier: MIT
+
+//TODO Add UR_RESULT_ERROR_INVALID_EVENT_WAIT_LIST negative test when this is implemented.
```

Comment:
I think this comment is unnecessary. The error code will still be covered in the description of the issue i.e. https://github.com/oneapi-src/unified-runtime/issues/235

Reply:
Fair enough!